ui-testing-core/src/main/java/com/avito/android/test/espresso/action/TextViewReadAction.kt

```diff
@@ -6,32 +6,40 @@ import android.support.test.espresso.matcher.ViewMatchers
 import android.view.View
 import android.widget.TextView
 import com.avito.android.test.waitFor
-import junit.framework.Assert
 import org.hamcrest.Matcher
+import org.junit.Assert.assertFalse
 
-class TextViewReadAction(private val allowBlank: Boolean) : ViewAction {
+class TextViewReadAction : ViewAction {
 
-    lateinit var result: String
-        private set
+    companion object {
+        fun getResultVia(allowBlank: Boolean, perform: (TextViewReadAction) -> Unit): String {
```

Comment:
Seems like method name is cut. I believe there should be `ViaSmth`

Reply:
hmm, yep, definitely a naming problem
maybe getResult with comments on parameters will be better